### PR TITLE
fireworks on success page

### PR DESCRIPTION
**Pull request by PiranavanShanmugavadivelu**
Issue - https://github.com/EmpowerPlastic/empower-platform/issues/5862

If you guys have any suggestions for different animations, You can find  [here](https://lottiefiles.com/search?q=fireworks&category=animations)

Current animation  - https://app.lottiefiles.com/animation/3f2098d5-9af2-4651-8476-df9db152751d?channel=web&source=public-animation&panel=download
image

### CODE REVIEW

**Review by ba1ciu — APPROVED** [review]
I like it :D

**Comment by jschill** [comment]
Haha lol sweet. I will take a look tomorrow. We have to make sure its possible to click the "See your certificates"-button and that the new lib wont add too much kbs

File: frontend/marketplace/src/main.ts [hunk 1]
```diff
@@ -51,6 +52,8 @@ app.provide(DefaultApolloClient, apolloClient);
 app.use(Vue3Toastify, {
   autoClose: 3000,
 } as ToastContainerOptions);
+app.use(Vue3Lottie);
```
Comment (jschill) [inline frontend/marketplace/src/main.ts:55]: Can you change from this global use in main.ts to a local use in the component instead? That way we can lazyload this when it's used.

**Review by jschill — CHANGES_REQUESTED** [review]
Sorry for late review

**Review by jschill — APPROVED** [review]
🔥 🚒 🧯 🧨 🧑‍🚒

**Pull request merged**